apps/ledger-live-desktop/tests/models/Drawer.ts

```diff
@@ -18,7 +18,7 @@ export class Drawer {
     this.currencyButton = (currency: string) =>
       page.locator(`data-test-id=currency-row-${currency.toLowerCase()}`);
     this.accountButton = (accountName: string, index: number) =>
-      page.locator(`data-test-id=account-row-${accountName.toLowerCase()}-${index}`);
+      page.locator(`data-test-id=account-row-${accountName.toLowerCase()}-${index}`).first();
```

Comment:
I saw one instance of a test failing because it found two of these. Since it's just one time it's probably flakiness, but this will make sure the element we want is clicked